test/index.test.ts

```diff
@@ -47,6 +49,13 @@ describe("Approveman tests", () => {
   });
 
   beforeEach(() => {
+    // Clears the jest cache so that the environment will
+    // not be contaminated between tests.
+    jest.resetModules();
```

Comment:
Codacy found an issue: ['jest' is not defined.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6168221)